src/test/java/org/broadinstitute/dsm/TestHelper.java

```diff
@@ -103,9 +106,6 @@ public static void setupDB(boolean setupDDPConfigLookup) {
         cfg = cfg.withValue("portal.port", ConfigValueFactory.fromAnyRef("9999"));
         cfg = cfg.withValue("errorAlert.recipientAddress", ConfigValueFactory.fromAnyRef(""));
 //
-        if (!cfg.getString("portal.environment").startsWith("Local")) {
```

Comment:
This might be controversial @SimoneMaiwald ... I found myself fiddling with local configs just to "cheat" and get around this check.

Reply:
but it made you double check your config =) so you would have hopefully realized when env was prod instead of dev :)

Reply:
I shall revert!